docs/sql-ref-syntax-ddl-create-sql-function.md

```diff
@@ -0,0 +1,305 @@
+---
+layout: global
+title: CREATE FUNCTION (SQL)
+displayTitle: CREATE FUNCTION (SQL)
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+The `CREATE FUNCTION` statement creates a SQL function that can be used in SQL statements. The function can be temporary or permanent, and can return either a scalar value or a table result. The function body can be defined either a SQL expression or a query.
+
+When `TEMPORARY` is specified, the function is only available for the current session. Otherwise, it is persisted in the catalog and available across sessions. The `OR REPLACE` option allows updating an existing function definition, while `IF NOT EXISTS` prevents errors when creating a function that already exists.
+
+The function parameters must be specified with their data types. The return type can be either a scalar data type or a table with an optional schema definition. 
```

Comment:
oh, the return table schema definition is optional?

Reply:
actually, the entire RETURN clause is optional, right?

Reply:
So it should be ` The return type can be either a scalar data type or a table with an schema definition. If not specified the return type will be inferred from the function body`?